lib/plaid/user.rb

```diff
@@ -212,7 +212,9 @@ def mfa_step(info = nil, send_method: nil, options: nil)
       conn = Connector.new(product, :step, auth: true)
 
       # Use PATCH if the {update: true} was passed in the options Hash
-      response = if options && options[:update]
+      use_patch = (options && options[:update].present?) ? options[:update] : @mfa_patch
```

Comment:
@gabrielgarza Unfortunately, we can't use `Object#present?` since it requires ActiveSupport. Furthermore, even if we had it, it wouldn't work the same as `not nil?`, since `false.present?` is `false`.